fuzzing/build.gradle.kts

```diff
@@ -13,5 +13,7 @@ dependencies {
 
 tasks.register<Copy>("copyToFuzzOut") {
     dependsOn(tasks.build)
-    into(project.property("fuzzOut")).from(sourceSets.main.get().runtimeClasspath)
+    val defaultDestination = project.layout.buildDirectory.dir("fuzzOut")
+    val dest = if (project.hasProperty("fuzzOut")) project.property("fuzzOut") ?: defaultDestination else defaultDestination
+    into(dest).from(sourceSets.main.get().runtimeClasspath)
```

Comment:
```suggestion
    into(dest)
    from(sourceSets.main.get().runtimeClasspath)
```

Reply:
is this more idiomatic?

Reply:
I think so
They describe two different things:
1) where to store the data
2) which data to pick


For instance, if you would ever add the second `from`, then `into(...).from(..).from(..)` would look too long, while the following is fine:

```
into(...)
from(...)
from(...)
```

Reply:
Consider this:

```kotlin
    into(dest)
    into("abcd") {
        from(sourceSets.main.get().runtimeClasspath)
    }
```

=>

```
% tree build/fuzzOut
build/fuzzOut
└── abcd
    ├── animal-sniffer-annotations-1.21.jar
    ├── annotations-4.1.1.4.jar
    ├── bcpkix-jdk18on-1.72.jar
    ├── bcprov-jdk18on-1.72.jar
```